cobalt/media/base/sbplayer_pipeline.cc

```diff
@@ -406,7 +454,13 @@ SbPlayerPipeline::SbPlayerPipeline(
                              pipeline_identifier_.c_str()),
           "", "The max video capabilities required for the media pipeline."),
       playback_statistics_(pipeline_identifier_) {
-  SbMediaSetAudioWriteDuration(kAudioLimit);
+  DCHECK(sbplayer_interface_);
+  if (!sbplayer_interface_->IsAudioWriteAheadExtensionEnabled()) {
+    SbMediaSetAudioWriteDuration(audio_write_duration_);
```

Comment:
Since it's on C23 and SbMediaSetAudioWriteDuration is always available, shall we always call it no matter if the extension is enabled?

Reply:
Done.